src/utils/form/createValidationSchema.ts

```diff
@@ -0,0 +1,65 @@
+import { z } from "zod";
+import { type Question } from "./types";
+import {
+  validateMultipleChoiceInput,
+  validateTextInput,
+} from "@/utils/form/validateInput";
+
+interface IField {
+  [key: string]:
+    | z.ZodString
+    | z.ZodEffects<z.ZodString, string, string>
+    | z.ZodArray<z.ZodString, "many">
+    | z.ZodArray<z.ZodEffects<z.ZodString, string, string>, "many">;
+}
+
+export function createValidationSchema(questionsData: Question[]) {
```

Comment:
> Is this only used for the sprint forms? If so, maybe a more specific name for this function would be better. It's too generic here.

I think it might be used for other forms as well in future.